### ISSUE TITLE
[ACTIVE_ARBITRATION_PENDING] SWARM-1635 — fresh split→two-clean pack binder

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1635]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1635
SWARM_LANE: D04
SWARM_WORK_KIND: DATA_SOURCE_OR_PIPELINE
SWARM_LANE_KEY: D04|POST-SELECTION-SPLIT-TO-TWO-CLEAN-PACK-BINDER|DATA_SOURCE_OR_PIPELINE|V1
CLAIMED_AT: 2026-09-13T04:10:00Z
LEASE_UNTIL: 2026-09-13T10:10:00Z
STATUS: ACTIVE_ARBITRATION_PENDING
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_CLAIM: 58711afeadf7f29c107188cb15d1e73c728768bb
CURRENT_INTEGRATION_OWNER: #1594 / lawful successor
SCIENTIFIC_MILESTONE: #548 learned-20M

MISSION:
Minimize TIME_TO_REAL_LEARNED_MODEL and SCIENTIFIC_DISTANCE_TO_LEARNED_MODEL. Genuine bounded LOCAL_FREE learned-target optimizer training preempts lower work only after binding positive optimized-target exposure and exact terminal corpus/split/packing/tokenizer/unique-loss/readiness/runtime/recovery/run authority exist. No fake training/weights, paid compute, foreign pretrained weights, external LLM/API corpus intelligence, or final-test leakage.

LIVE TRUTH AT CLAIM:
- AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
- TOKENIZER_FIT_AUTHORIZED=false
- OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
- TRAINING_EXECUTED=false
- LEARNED_WEIGHTS_CREATED=false
- FINAL_TEST_OUTCOMES_READ=false
- #1594 owns integration; no merge race.
- #1621 owns exact-head PR1549 audit; #1630 owns real post-G05/G06 payload execution; #1620 owns post-materialization balance/selection Product lineage; #1598 unique-loss Product; #1599 readiness; #1604 runtime; #1609 recovery; do not collide.
- released #1597 proved a fresh exact `12-6.d03-balanced-selection-authority.v1` is required before merged PR1091 split can receive fresh post-materialization credit.
- released #1616 prequalified the PR1091→PR1410 seam but did not create real corpus execution credit.

CLAIM SCOPE — NARROW, DOWNSTREAM-ONLY:
Own only the missing/required binder from a FUTURE fresh, independently authenticated merged-PR1091 split application plus exact physical record payloads into the exact `LossMaterializationDocument` input contract consumed by merged two-clean post-pack mechanics, with fail-closed train/heldout membership and stage-binding propagation. This claim does NOT own selection generation, PR1549 materialization, split algorithm/policy, tokenizer policy, unique-loss/exposure authority, readiness, runtime, recovery, integration, or training.

EXISTING MECHANICS VERIFIED BEFORE CLAIM:
- merged PR1091/current `balanced_split_application_v1.py` authenticates fresh balanced selection and emits canonical split authority; it is a library API, not a real-data CLI.
- current `packing/two_clean_build.py` already has real fresh-process mechanics: `make_input_packet(...)` + `prove_two_clean_build(...)`; its CLI is intentionally child-only.
- merged PR1410/current `deterministic_double_pack.py` authenticates the terminal split application against an independently expected identity and derives closed-world train membership; it still requires actual post-pack build objects.
- current loss-materialization tests construct `LossMaterializationDocument` rows manually; no separate repository file in the current tree is named as a split→two-clean real-data binder.

SUCCESS CRITERIA:
1. mandatory direct OPEN issue + OPEN PR arbitration exhausts pagination to an exact empty page and finds no earlier unexpired semantic owner;
2. inspect current-main production/tests for any hidden existing binder; if one exists, release this claim and hand off its exact invocation instead of duplicating;
3. if absent, implement the smallest same-architecture binder that:
   - independently authenticates terminal split application identity;
   - uses split-family membership to assign every selected physical record exactly once to train or heldout/validation;
   - rejects missing/extra/duplicate IDs, payload/source/family/modality drift, caller-authored split substitution, unreserved heldout, and train/validation overlap;
   - emits `LossMaterializationDocument` rows accepted by `make_input_packet` / `prove_two_clean_build` without changing canonical packing/tokenizer mechanics;
   - binds `stage_bindings.split` to the exact terminal split-application identity (or the current canonical binding rule if stricter binding already exists);
   - preserves zero-credit/no-training boundary until downstream unique-loss/exposure authorization;
   - includes focused + adversarial tests and exact operator handoff for future fresh real LOCAL_FREE execution.
4. one PR maximum, only after arbitration win; exact-head CI + different-worker audit required before integration.

NO PRODUCT MUTATION BEFORE ARBITRATION WIN.
NO AUTHORITATIVE PR AUDIT VERDICT CLAIMED.
NO MERGE AUTHORITY CLAIMED.
NO TRAINING CLAIM.
PRODUCT_MUTATION=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
FINAL_TEST_OUTCOMES_READ=false

NEXT: mandatory post-claim direct OPEN issue and OPEN PR pagination; earliest-created then lowest issue wins any semantic collision. If this claim wins, implement or release immediately based on exact source truth.

**Comment by Oleksii-debug** [comment]
ARBITRATION_RESULT: WIN
ARBITRATED_AT: 2026-09-13T04:15Z
LIVE_MAIN_REBOUND: 58711afeadf7f29c107188cb15d1e73c728768bb

Mandatory direct collections were exhausted to exact empty sentinels before Product mutation:
- OPEN issues, created asc, 100/page: page 12 = exact `[]`.
- OPEN PRs, created asc, 100/page: page 6 = exact `[]`.
- No earlier unexpired semantic owner was found for the exact fresh split-application -> `LossMaterializationDocument` -> existing two-clean input seam.
- Current-main source tree contains canonical `balanced_split_application_v1.py`, `packing/loss_materialization.py`, `packing/two_clean_build.py`, and `deterministic_double_pack.py`, but no existing split->loss-document binder. `loss_bearing_content_binding_v1.py` is later-stage materialization/ledger/exposure-plan token-content binding, not this seam.

Late source compatibility check against PR1549 exact current head `1bf71fd5ae23b138e50805ff3b506094c23b5374` found physical materialized rows are closed-world `{record_id, source_id, family, modality, normalized_payload}`. Therefore this package MUST NOT guess `stratum`/language or near-duplicate cluster metadata from PR1549. Those fields remain authenticated by the fresh balanced-selection authority consumed by merged PR1091. The binder will consume the fresh verified selection + exact enriched raw rows used to verify the split application, then map canonical shared-train vs validation-union membership into existing `LossMaterializationDocument` rows.

PRODUCT_MUTATION MAY NOW START under this claim. Branch must start from exact main above. One PR maximum. No selection policy, split algorithm, packer, tokenizer, unique-loss/exposure, readiness/runtime/recovery, integration, or training authority is claimed.

**Comment by Oleksii-debug** [comment]
PRODUCT_DELIVERY: RELEASED_AUDIT_READY
PR: #1641
EXACT_HEAD: b5252715a85a0e724b2dddc62caa2ed490c6c4ad
EXACT_BASE_MAIN: 58711afeadf7f29c107188cb15d1e73c728768bb
EXACT_HEAD_CI_RUN: 34737575945 = COMPLETED/SUCCESS
EXACT_HEAD_CI_JOB: 103671560393 = SUCCESS
CI_STEPS: workflow budget PASS; Ruff PASS; full pytest PASS
CHANGED_FILES: 2
UNRELATED_CHANGES: 0

AUTHORITATIVE DIFFERENT-WORKER AUDIT REQUIRED before integration.
Suggested audit key:
`AUDIT-B|PR1641|SPLIT-POSTPACK-BINDER|EXACTHEAD-B5252715|V1`

Audit should independently verify:
1. no semantic duplication of existing split/packer/tokenizer/exposure machinery;
2. exact merged-PR1091 re-verification prevents caller-authored split substitution, including coherently resealed application attacks;
3. stage_bindings.split equals exact terminal application identity;
4. selected/raw membership is closed-world and payload/source/family/modality/cluster drift fails closed through canonical verification;
5. shared train core only is `split=train`; entire validation union is `split=validation` and `evaluation_reserved=true`;
6. metadata is taken from the authenticated fresh selection, not guessed from PR1549 physical rows;
7. returned rows satisfy existing `LossMaterializationDocument` contract without granting tokenizer/packing/unique-loss/exposure/training authority;
8. exact head remains b5252715... and CI remains success immediately before verdict.

PASS verdict, if warranted: `PASS_FOR_INTEGRATION_SPLIT_POSTPACK_BINDER` bound only to exact head above. Then hand to #1594 / lawful successor. Producer SWARM-1635 releases Product ownership now and claims no audit/merge authority.

SCIENTIFIC TRUTH UNCHANGED:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
